hooks/info

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu
+vault="secret/$GENESIS_VAULT_PREFIX"
+
+ip=$(lookup params.ip)
+port=$(lookup params.port 3000)
+export SB_BROKER_URL=http://$ip:$port
+export SB_BROKER_USERNAME=blacksmith
+export SB_BROKER_PASSWORD=$(safe read $vault/broker:password)
+
+export BOSH_ENVIRONMENT=https://$(lookup params.ip):25555
+export BOSH_CA_CERT=$(safe read $vault/tls/ca:certificate)
+export BOSH_CLIENT=admin
+export BOSH_CLIENT_SECRET=$(safe read $vault/users/admin:password)
+
+
+describe "ca certificate"
+         safe read $vault/tls/ca:certificate
+    echo
+    echo "bosh env"
+         bosh env --tty | sed -e 's/^/  /';
+    echo
+    echo
+    echo "blacksmith (internal) bosh director"
```

Comment:
from what I understand, blacksmith can be configured to use the same BOSH director as it was deployed to.  Though this is not recommended, the `info` hook does not seem to take this into consideration.  Is this no longer a possibility, or considered so out of the norm that if you've done this, you and everyone else with access to the deployment repo will know that its been done?

Reply:
Blacksmith can do it; the kit cannot.  As an enforcer of best practices, the kit does not give the option to target anything other than the internal BOSH director.